test/interaction/template.html

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
```

Comment:
It is unnecessary to include this file. See [minimal webpack example](https://github.com/uber/deck.gl/tree/master/examples/get-started/react-webpack-2).